.travis.yml

```diff
@@ -1,16 +1,18 @@
 language: node_js
 node_js:
 - 6.6.0
+sudo: false
 before_install:
-- npm install -g mocha
-- npm install -g gulp
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
I was under that impression too, Travis was still using npm for the build prior to this commit _possibly as yarn.lock isn't merged into master?_